vue-app/src/views/Profile.vue

```diff
@@ -153,22 +159,9 @@ export default class NavBar extends Vue {
     return CHAIN_INFO[Number(process.env.VUE_APP_ETHEREUM_API_CHAINID)]
   }
 
-  renderUserAddress(digitsToShow?: number): string {
```

Comment:
Don't we want to use this if we're displaying an address? I think this change created this issue: https://github.com/ethereum/clrfund/issues/327

Reply:
@samajammin So I tried switching this out for just using CSS to end with an ellipsis, allowing some more flexibility with what gets placed here, and it's just simpler to deal with that this helper function. 

In issue #327, this is all due to not rounding the decimals to 5 in this component, and can be addressed separately. Don't this directly caused that issue. 

If there is a stylistic preference to not let the 0x addresses get too long, we can still place a character limit on this element to start the ellipsis sooner. Let me know what you think, otherwise I'll leave as it.

Reply:
Ah good call, didn't see the CSS change 👍